migration/migration.go

```diff
@@ -76,8 +86,10 @@ var defaultMigrations = []Migration{
 
 var ErrCallWithNewTransaction = errors.New("call with new transaction")
 
-func MigrateIfNeeded(ctx context.Context, targetDB db.KeyValueStore, network *utils.Network, log utils.SimpleLogger) error {
-	return migrateIfNeeded(ctx, targetDB, network, log, defaultMigrations)
+func MigrateIfNeeded(ctx context.Context, targetDB db.KeyValueStore, network *utils.Network,
+	log utils.SimpleLogger, httpConfig HTTPConfig,
+) error {
+	return migrateIfNeeded(ctx, targetDB, network, log, defaultMigrations, httpConfig)
 }
```

Comment:
Let's pass `httpConfig` by reference here. Example
```suggestion
func MigrateIfNeeded(ctx context.Context, targetDB db.KeyValueStore, network *utils.Network,
	log utils.SimpleLogger, httpConfig *HTTPConfig,
) error {
	return migrateIfNeeded(ctx, targetDB, network, log, defaultMigrations, httpConfig)
}
```

The same change should be applied to `migrateIfNeeded`.

Then, every other function call to this should add the `&` operator to make it a reference.

```go
err := MigrateIfNeed(ctx, tragetDB, network, logger, &httpconfig)
```

Reply:
done